src/UI/TextField.elm

```diff
@@ -785,3 +774,88 @@ textFieldPadding size =
 
         Size.ExtraSmall ->
             Element.paddingXY 8 7
+
+
+nonStaticView : RenderConfig -> Properties msg -> Options msg -> (String -> msg) -> Element msg
+nonStaticView cfg prop opt msg =
+    let
+        elAttrs =
+            attrs cfg prop opt
+
+        whenPassword { isVisible, isCurrent } =
+            if isCurrent then
+                inputPasswordOptions cfg msg prop opt isVisible
+                    |> Input.currentPassword elAttrs
+
+            else
+                inputPasswordOptions cfg msg prop opt isVisible
+                    |> Input.newPassword elAttrs
+    in
+    case prop.content of
+        ContentSinglelineText ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.text elAttrs
+
+        ContentMultilineText ->
+            inputMultilineOptions cfg msg prop opt
+                |> Input.multiline elAttrs
+
+        ContentUsername ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.username elAttrs
+
+        ContentPassword pswOpt ->
+            whenPassword pswOpt
+
+        ContentEmail ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.email elAttrs
+
+        ContentSearch ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.search elAttrs
+
+        ContentSpellChecked ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.spellChecked elAttrs
+
+
+staticView : RenderConfig -> Properties msg -> Options msg -> Element msg
+staticView renderConfig prop opt =
+    let
+        elAttrs =
+            attrs renderConfig prop opt
+
+        genericInputElement value =
+            value
+                |> Text.subtitle2
+                |> Text.renderElement renderConfig
+                |> Element.el elAttrs
+
+        inputElement =
+            case prop.content of
+                ContentPassword _ ->
+                    "●"
+                        |> String.repeat (String.length prop.currentValue)
+                        |> genericInputElement
```

Comment:
Why not just use the password input type?

Reply:
elm-ui docs recommend this: https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element-Input#disabling-inputs

Reply:
> For other inputs such as Input.text, consider simply rendering it in a normal paragraph or el if it's not editable.
Alternatively, see if it's reasonable to not display an input if you'd normally disable it. Is there an option where it's only visible when it's editable?

Reply:
Hmm not sure this applies to the input case, the discussion linked in the documentation is about a button that allows the user to progress. After reading it through I actually believe now that our "fake" disabled behavior could be bad, because there's nothing but the visual cue to indicate that it is disabled, so it could be  interpreted as not working

Reply:
I agree with you, it doesn't seem that having a field that's "always disabled" will be useful.

As this is impossible in elm-ui, by now let's go with killing the `static` initializer, but reusing it in a `withDisabledIf`